### DISCUSSION TITLE
Adding the ability to select a MongoDB/MongoDB Atlas database as datasource

### DISCUSSION THREAD

**Discussion by MazeWebArchitect** [TiagoSilvaPereira/vemto-issues#156] [Ideas & Suggestions]
**Is your feature request related to a problem? Please describe.**
I'd like to connect a MongoDB/MongoDB Atlas Database as main datasource for some Laravel projects.

**Describe the solution you'd like**
In the setup dialog, I'd like to have the option to select a MongoDB/MongoDB Atlas Database.

**Describe alternatives you've considered**
Right now I have to setup a mock MySQL Database and change my setup after generating the code via Vemto.

**Additional context**
Maybe there is a possible implementation method within this repo: https://github.com/jenssegers/laravel-mongodb
This is the repo I clone and overwrite right now after generating my app via Vemto.

**Comment by TiagoSilvaPereira** [comment]
Hi @MazeWebArchitect thank you for your suggestion... I'm moving it to the discussion tab
